electroncash_gui/qt/utxo_list.py

```diff
@@ -32,7 +33,6 @@
 from functools import wraps
 from enum import IntEnum
 
-
```

Comment:
nit: https://peps.python.org/pep-0008/#blank-lines

Reply:
Done in https://github.com/scinklja/Electron-Cash/commit/98177658d51b4e2d85147bccd20d5f35f792bea5